cmd/bridge/main.go

```diff
@@ -57,6 +57,9 @@ const (
 
 	// Well-known location of metering service for OpenShift. This is only accessible in-cluster.
 	openshiftMeteringHost = "reporting-operator.openshift-metering.svc:8080"
+
+	// Well-known location of the GitOps service. This is only accessible in-cluster
+	gitOpsBackendHost = "cluster.pipelines-app-delivery.svc:8080"
```

Comment:
Can this be installed in a reserved, `openshift-` prefixed namespace?

Reply:
Sounds good, I would go for that in the operator.

Reply:
Done. @chetan-rns Could you update your PR to reflect the change 
https://github.com/redhat-developer/gitops-operator/blob/master/pkg/controller/gitopsservice/gitopsservice_controller.go#L35